src/libraries/System.Private.CoreLib/src/System/IO/MemoryStream.cs

```diff
@@ -311,7 +312,7 @@ public override long Position
                 EnsureNotClosed();
 
                 if (value > MemStreamMaxLength - _origin)
-                    throw new ArgumentOutOfRangeException(nameof(value), SR.ArgumentOutOfRange_StreamLength);
+                    throw new ArgumentOutOfRangeException(nameof(value), SR.Format(SR.ArgumentOutOfRange_StreamLength, Array.MaxLength));
```

Comment:
I don't have a strong preference to how we format the exception message cc @tannergooding.